sites.js

```diff
@@ -0,0 +1,179 @@
+	Postimees: 'postimees.ee',
+	PZC: 'pzc.nl',
+	'Quartz (free articles only)': 'qz.com',
+	Quora: 'quora.com',
+	'Quotidiani Gelocal': 'gelocal.it',
+	'Republic.ru': 'republic.ru',
+	Reuters: 'reuters.com',
+	'San Diego Union Tribune': 'sandiegouniontribune.com',
+	'San Francisco Chronicle': 'sfchronicle.com',
+	'Scientific American': 'scientificamerican.com',
+	'Seeking Alpha': 'seekingalpha.com',
+	Slate: 'slate.com',
+	SOFREP: 'sofrep.com',
+	'Star Tribune': 'startribune.com',
+	Statista: 'statista.com',
+	Stuff: 'stuff.co.nz',
+	'Süddeutsche Zeitung': 'sueddeutsche.de',
+	SunSentinel: 'sun-sentinel.com',
+	'Tech in Asia': 'techinasia.com',
+	Telegraaf: 'telegraaf.nl',
+	Time: 'time.com',
+	'The Advertiser': 'adelaidenow.com.au',
+	'The Advocate': 'theadvocate.com.au',
+	'The Age': 'theage.com.au',
+	'The American Interest': 'the-american-interest.com',
+	'The Athletic': 'theathletic.com',
+	'The Athletic (UK)': 'theathletic.co.uk',
+	'The Atlantic': 'theatlantic.com',
+	'The Australian Financial Review': 'afr.com',
+	'The Australian': 'theaustralian.com.au',
+	'The Business Journals': 'bizjournals.com',
+	'The Canberra Times': 'canberratimes.com.au',
+	'The Courier': 'thecourier.com.au',
+	'The Courier Mail': 'couriermail.com.au',
+	'The Cut': 'thecut.com',
+	'The Daily Telegraph': 'dailytelegraph.com.au',
+	'The Diplomat': 'thediplomat.com',
+	'The Economist': 'economist.com',
+	'The Globe and Mail': 'theglobeandmail.com',
+	'The Herald': 'theherald.com.au',
+	'The Hindu': 'thehindu.com',
+	'The Irish Times (free articles only)': 'irishtimes.com',
+	'The Japan Times': 'japantimes.co.jp',
+	'The Kansas City Star': 'kansascity.com',
+	TheMarker: 'themarker.com',
+	'The Mercury News': 'mercurynews.com',
+	'The Mercury Tasmania': 'themercury.com.au',
+	'The Morning Call': 'mcall.com',
+	'The Nation': 'thenation.com',
+	'The National': 'thenational.scot',
+	'The News-Gazette': 'news-gazette.com',
+	'The New Yorker': 'newyorker.com',
+	'The New York Times': 'nytimes.com',
+	'The Olive Press': 'theolivepress.es',
+	'The Philadelphia Inquirer': 'inquirer.com',
+	'The Saturday Paper': 'thesaturdaypaper.com.au',
+	'The Seattle Times': 'seattletimes.com',
+	'The Spectator Australia': 'spectator.com.au',
+	'The Spectator (U.K.)': 'spectator.co.uk',
+	'The Spectator (U.S.)': 'spectator.us',
+	'The Sydney Morning Herald': 'smh.com.au',
+	'The Telegraph': 'telegraph.co.uk',
+	'The Toronto Star': 'thestar.com',
+	'The Wall Street Journal': 'wsj.com',
+	'The Washington Post': 'washingtonpost.com',
+	'The Wrap': 'thewrap.com',
+	'Times Literary Supplement': 'the-tls.co.uk',
+	'Towards Data Science': 'towardsdatascience.com',
+	Trouw: 'trouw.nl',
+	Tubantia: 'tubantia.nl',
+	'Vanity Fair': 'vanityfair.com',
+	'Vrij Nederland': 'vn.nl',
+	Vulture: 'vulture.com',
+	'Winston-Salem Journal': 'journalnow.com',
+	Wired: 'wired.com',
+	'World Politics Review': 'worldpoliticsreview.com',
+	'Zeit Online': 'zeit.de',
+	'*General Paywall Bypass*': 'generalpaywallbypass',
```

Comment:
Do we need this?

Reply:
Oh probably not. Probably should have read what I was copy-pasting 😅

Reply:
Pushed up a fix